### ISSUE TITLE
Code inspection result "Object variable 'foo' is assigned without 'Set' keyword"

### ISSUE THREAD

**Issue by SmileyFtW** [rubberduck-vba/Rubberduck#5023]
I've been implementing some concepts I read on the WordPress site and one in particular:

```
Public Property Get Foo () as Object
   Set Foo = this.foo
End Property
Public Property Let Foo (byval obj as Object)
   Set this.foo = obj
End Property
```

This arrangement throws the subject code inspection.
Is using the Set inside of Property Get/Let the wrong way to do it?
Thanks!
Miley

**Comment by retailcoder** [comment]
That should be a `Property Set` member, but the inspection is confused regardless (a number of issues are open for it), presumably because of the UDT-member nature of `foo` here.

**Comment by SmileyFtW** [comment]
Thanks, Mathieu.... then I'd suggest saying something about it in the examples on the WP site. Specifically the "There is no workbook" and related posts that use that convention... I'll change my Lets to Sets for objects.
On a side note, I wrote something over the weekend for a friend's golf outing using most of what I've learned from the RD community. After the project was "in the can" and ready to go; he let me know, "Oh yeah I forgot to tell you I need it to do "this" too..." Whoops. Well, because of the structured way you've taught me, the "new stuff" took lest than an hour (more like 20-30 minutes) to implement and test. Friggen' AWESOME!
